R4.2/qubes-os-r4.2-templates-community.yml

```diff
@@ -49,7 +49,7 @@ components:
   - template-kali:
       packages: False
       fetch-versions-only: false
-      branch: main
+      branch: master
```

Comment:
No "main" branch there? If you create one, we have a bot that will keep main in sync with master.